robolectric/src/test/java/org/robolectric/shadows/ShadowAccountManagerTest.java

```diff
@@ -1130,4 +1133,59 @@ T getResult() throws Exception {
       return accountManagerFuture.getResult();
     }
   }
+
+  @Test
+  @Config(minSdk = O)
+  public void
+      accountManager_activityContextEnabled_applicationInstanceIsNotSameAsActivityInstance() {
```

Comment:
If this test is already on ctesque tests, it does not need to be here. ctesque is a more reliable location.